teku.rb

```diff
@@ -5,7 +5,7 @@ class Teku < Formula
   # update with: ./updateTeku.sh <new-version>
   sha256 "07c547f79cf8502ebed0469440ea2c577e420dbb0a374a7ff2680fff38e34be4"
 
-  depends_on :java => "11+"
+  depends_on :openjdk => "11+"
```

Comment:
The `:openjdk` syntax is looking for a special package name so it needs to be:
```suggestion
  depends_on "openjdk" => "11+"
```

Reply:
Works with both versions, will apply suggestion.

Reply:
hmm, interesting - it failed when I tried it with the original version.